logistics_project/apps/malawi/tests/util_functions.py

```diff
@@ -136,13 +138,26 @@ class TestErrorSwallowing(TestCase):
     def test_swallow(self):
         class TestError(Exception): pass
 
-        def _fail_if_even(val):
+        def _fail_if_even(val, error_type):
             if val % 2 == 0:
-                raise TestError('No even numbers allowed!')
+                raise error_type('No even numbers allowed!')
 
         x = []
         for i in range(5):
             with swallow_errors(TestError):
-                _fail_if_even(i)
+                _fail_if_even(i, TestError)
                 x.append(i)
         self.assertEqual([1, 3], x)
+
+        # test default behavior
+        x = []
+        for i in range(5):
+            with swallow_errors():
+                _fail_if_even(i, MessageSendingError)
+                x.append(i)
+        self.assertEqual([1, 3], x)
+
+        # test wrong type
+        with self.assertRaises(TestError):
+            with swallow_errors():
+                _fail_if_even(i, TestError)
```

Comment:
small thing but it might be better to explicitly pass an even number here rather than use `i` since someone could modify this test at a later point, using `i` to iterate again and then not realize they're changing the outcome of this check below

Reply:
oh wow, good catch, definitely a copy/paste error that just happened to work.

Reply:
https://github.com/dimagi/logistics/pull/287/commits/44147a388d2a63f062944c027ad3151d5f03cfef